config.js

```diff
@@ -4,7 +4,7 @@ var config = {
   twitter: (process.env.TTC_BOTS || 'tinycarebot,selfcare_bot,magicrealismbot').toLowerCase().split(','),
 
   // Use this to have a different animal say a message in the big box.
-  say: (process.env.TTC_SAY_BOX || 'parrot').toLowerCase(),
+  say: (process.env.TTC_SAY_BOX || 'random').toLowerCase(),
```

Comment:
Please keep the default to be `parrot`, not `random`. I, for example, don't set this property, and would absolutely not want a random animal :)